index.html

```diff
@@ -471,6 +471,11 @@ <h2>NEL policies</h2>
       when the user agent received the policy.
       </p>
 
+      <p>
+      A <a>NEL policy</a> is <dfn>stale</dfn> if its <a>creation</a> plus
+      172800 seconds (48 hours) represents a time in the past.

```

Comment:
Should we define this using hr-time algorithms? (E.g. current wallclock time)

Reply:
Done -- updated the handling of creation, stale and expired to use current wall time from HR-TIME.